File: InvoiceCalculatorJava/src/main/java/com/uniandes/InvoiceCalculatorJava/dto/InvoiceResponse.java
package com.uniandes.InvoiceCalculatorJava.dto;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;
import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class InvoiceResponse {

    private List<InvoiceItemResponse> items;
    private Double total;
}

File: InvoiceCalculatorC/InvoiceCalculatorC/Data/Dto/InvoiceResponse.cs
using InvoiceCalculatorC.Models;

namespace InvoiceCalculatorC.Data.Dto;

public class InvoiceResponse
{
    public List<Product> items { get; set; }
    public double total { get; set; }

    public InvoiceResponse(List<Product> items, double total)
    {
        this.items = items;
        this.total = total;
    }
}